website/src/views/OrganismConstants.ts

```diff
@@ -9,12 +9,24 @@ export interface OrganismConstants {
     readonly dataOrigins: DataOrigin[];
 }
 
+export const ComponentHeight = {
+    large: '600px',
+    small: '300px',
+} as const;
```

Comment:
Maybe we should have a section on the page with smaller and larger elements, or refactor it, that the elements can have different sizes. (Not this PR)

Reply:
We should also use these constants on the other "Generic" pages.

Reply:
> We should also use these constants on the other "Generic" pages.

This will be part of my next PR, so I'll ignore this here.